### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by kant**
(proposals)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -275,8 +275,8 @@ It is possible to use travis-ci in order to build and upload your tagged release
 	```
 1. Run `travis setup releases`, follow documentation [here](https://docs.travis-ci.com/user/deployment/releases/)
 
-	This command will encode your github credentials into the .travis.yml file in order to let travis upload the release to github.com
-	When prompted for the file to upload, enter $FRAMEWORK_NAME.framework.zip
+	This command will encode your Github credentials into the `.travis.yml` file in order to let travis upload the release to Github.com
```
Comment (mdiep) [inline README.md]: This should really be `GitHub` with a capital _H_. Would you mind fixing that minor detail?

File: README.md [hunk 2]
```diff
@@ -123,17 +123,17 @@ After you’ve finished the above steps and pushed your changes, other users of
...
+In the Test target under the `Build Settings` tab, add `@loader_path/Frameworks` to the `Runpath Search Paths` if it isn't already present.
 
 In rare cases, you may want to also copy each dependency into the build product (e.g., to embed dependencies within the outer framework, or make sure dependencies are present in a test bundle). To do this, create a new “Copy Files” build phase with the “Frameworks” destination, then add the framework reference there as well.
 
 ### Upgrading frameworks
 
 If you’ve modified your [Cartfile][], or you want to update to the newest versions of each framework (subject to the requirements you’ve specified), simply run the `carthage update` command again.
 
-If you only want to update one , or specific, dependencies, pass them as a space-separated list to the `update` command. e.g.
+If you only want to update one, or specific, dependencies, pass them as a space-separated list to the `update` command. e.g.
```
Comment (jdhealy) [inline README.md]: Nice catch! :+1:

**Comment by mdiep** [comment]
> Stylistically, the “” → `` change looks nice, but, semantically, the HTML tag <code> mostly makes sense for source code and terminal output, rather than quotation of other on-screen content.

Maybe use _italics_ instead?

**Comment by blender** [comment]
Hi @kant , thanks for you contribution! Any plans to implement the requested changes?

**Comment by blender** [comment]
@kant ping

**Comment by blender** [comment]
Thanks for you contribution @kant, your commits are part of #2356 👍

**Pull request closed without merging**